workflow/cron/controller.go

```diff
@@ -84,6 +85,8 @@ func (cc *Controller) Run(ctx context.Context) {
 	})
 	cc.addWorkflowInformerHandler()
 
+	cc.wfLister = util.NewWorkflowLister(cc.wfInformer)
```

Comment:
check to see if this informer contains both completed and in-complete workflows -  I seem to recall it contained only in-complete versions

Reply:
The informer options are defined right above and contains all workflows with `LabelKeyCronWorkflow`